tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+            should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
+            testCredential({credential: issuedVc});
+          });
+          it('If present, the id value is expected to be a URL ' +
+            'that identifies the status information associated ' +
+            'with the verifiable credential. ' +
+            'It MUST NOT be the URL for the status list.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20the%20value%20is%20expected%20to%20be%20a%20URL%20that%20identifies%20the%20status%20information%20associated%20with%20the%20verifiable%20credential.';
+            if(!!issuedVc.credentialStatus.id) {
+              // TODO test for URI
+              issuedVc.credentialStatus.id.should.be.a('string',
+                'Expected credentialStatus.id to be a string.'
+              );
+            } else {
+              this.skip();
+            }
+          });
+          it('The type property MUST be BitstringStatusListEntry. ',
+            async function() {
+              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20type%20property%20MUST%20be%20BitstringStatusListEntry.';
+              issuedVc.credentialStatus.should.have.own.property(
+                'type').to.be.a('string',
+                'Expected credentialStatus.type to be a string.'
+              );
+              issuedVc.credentialStatus.type.should.equal(
+                'BitstringStatusListEntry',
+                'Expected credentialStatus.type to be BitstringStatusListEntry.'
+              );
+            });
+          it('The purpose of the status entry MUST be a string.',
+            async function() {
+              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20purpose%20of%20the%20status%20entry%20MUST%20be%20a%20string';
+              issuedVc.credentialStatus.should.have.own.property(
+                'statusPurpose').to.be.a('string',
+                'Expected credentialStatus.statusPurpose to be a string.'
+              );
+            });
+          it('The statusListIndex property MUST be an arbitrary size integer ' +
+            'greater than or equal to 0, expressed as a string in base 10.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20statusListIndex%20property%20MUST%20be%20an%20arbitrary%20size%20integer%20greater%20than%20or%20equal%20to%200%2C%20expressed%20as%20a%20string%20in%20base%2010';
+            issuedVc.credentialStatus.should.have.own.property(
+              'statusListIndex').to.be.a('number',
+              'Expected credentialStatus.statusListIndex to be an integer.'
+            );
+            issuedVc.credentialStatus.statusListIndex.should.be.gte(0,
+              'Expected credentialStatus.statusListIndex to be >= 0.');
+            //   TODO check for base 10
```

Comment:
that's not going to be easy, but I suppose making sure it's not hex could be an option? so no `0xF2` and such.

Reply:
could also try seeing if `Integer.parse(index) == index` or something like that.

Reply:
I settled for this:
```
assert(
  String(parseInt(issuedVc.credentialStatus.statusListIndex)) ===
  issuedVc.credentialStatus.statusListIndex
)
```